Playbooks/Linux setup.yml

```diff
@@ -1,31 +1,25 @@
 - name: Extend the logical volume to take all remaining space of the PVs and resize the underlying filesystem
   hosts: "*"
-  become: yes
+  become: true
```

Comment:
Both work, but as of late the recommendation is to use `true` instead of `yes`. Moreover, it is good to be consistent throughout your code, so not a good practice to use both.